main.yaml

```diff
@@ -149,6 +150,7 @@ tags:
         Glossaries are available in the <a href="https://phrase.com/pricing/" target="_blank">Pro Plan and higher</a>.
       </div>
   - name: Bitbucket Sync
+  - name: GitHub Sync
```

Comment:
You already wanted to add `GitHub Sync` in this PR?

Reply:
🤦 yep, I will remove it from this PR